streamparse/storm/bolt.py

```diff
@@ -300,10 +304,10 @@ def process_batch(self, key, tups):
     Note: Can be fractional to specify greater precision (e.g. 2.5).
     """
 
-    def __init__(self):
-        super(BatchingBolt, self).__init__()
+    def __init__(self, *args, **kwargs):
+        super(BatchingBolt, self).__init__(*args, **kwargs)
         self.exc_info = None
-        signal.signal(signal.SIGINT, self._handle_worker_exception)
+        signal.signal(signal.SIGUSR1, self._handle_worker_exception)
```

Comment:
I was running tests and got irritated that CTRL-C would cause an exception in `_handle_exception`, because `self.exc_info` wouldn't be set.  It made me realize that what we really wanted was a signal that didn't conflict with anything else.